bokeh/embed/bundle.py

```diff
@@ -414,7 +414,16 @@ def _use_mathjax(objs: Sequence[Model | Document]) -> bool:
         bool
     '''
     from ..models.text import MathText
-    return _any(objs, lambda obj: isinstance(obj, MathText))
+
+    def model_require_mathjax(model: Model) -> bool:
+        ''' If any model explicit require MathJax with render_as_mathtext option set to true
+        '''
+
+        for key, value in model._property_values.items():
```

Comment:
Iterating over every property on every model seems potentially expensive, so while I wouldnt usually recommend getattr I think I'd use some variation of `getattr(model, "render_as_mathtext", False)` here.

Reply:
Just updated it, is this similar to what you are saying? 

https://github.com/bokeh/bokeh/blob/1d8460aab11b858326be8c7a379e76d936a85d5d/bokeh/embed/bundle.py#L418-L424

Reply:
That seems totally fine, I probably would have just done: `return  properties.get('enable_mathtext', False)`

Reply:
That looks clean! Thanks!